packages/libp2p-interfaces/src/pubsub/index.ts

```diff
@@ -95,7 +95,7 @@ export interface PubSub extends EventEmitter {
   subscribe: (topic: string) => void
   getSubscribers: (topic: string) => string[]
   unsubscribe: (topic: string) => void
-  publish: (topic: string, data: Uint8Array) => Promise<void>
+  publish: (topic: string, data: Uint8Array) => Promise<Set<string>>
```

Comment:
I think it makes more sense to just return `number` because we don't really care _which_ specific nodes we publish too. Libp2p rust also returns number instead of the set of peers.